docs/content/development/extensions-core/caffeine-cache.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: doc_page
+---
+
+Druid Caffeine Cache
+--------------------
+
+A highly performant local cache implementation for Druid based on [Caffeine](https://github.com/ben-manes/caffeine). Requires a JRE with a fix for https://bugs.openjdk.java.net/browse/JDK-8078490
```

Comment:
can we add a note caution and just mention this extension only works for java 8?